tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md

```diff
@@ -7,8 +7,3 @@ Please follow this format for the actual expected failures.
 Level-3 headings should be used for the references to the relevant issues. Include the issue title with a link to the issue in GitHub.
 
 Other free text and markdown formatting can be used elsewhere in the document if needed. But if you want to explain something about the issue, then please post that in the issue itself.
-
-### [Exit page re-appears in loop when logged in user is deleted](https://github.com/owncloud/web/issues/4677)
-
-- [webUILogin/openidLogin.feature:50](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUILogin/openidLogin.feature#L50)
-- [webUILogin/openidLogin.feature:60](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUILogin/openidLogin.feature#L60)
```

Comment:
do we have the test coverage for this bug report in unit/e2e/api? just to be sure that we are not decreasing the coverage

CC @ScharfViktor

Reply:
It is a web issue so I think adding coverage in API won't do it.

Reply:
yes, but it's low priority bug. I wouldn't write tests for the p4 prio bugs